2022-02-10 15:59:46,526 INFO    MainThread:603371 [wandb_setup.py:_flush():71] setting env: {}
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_155946-1yzrnelx/logs/debug.log
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220210_155946-1yzrnelx/logs/debug-internal.log
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_init.py:init():404] calling init triggers
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-10 15:59:46,527 INFO    MainThread:603371 [wandb_init.py:init():460] starting backend
2022-02-10 15:59:46,527 INFO    MainThread:603371 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-10 15:59:46,537 INFO    MainThread:603371 [backend.py:ensure_launched():216] starting backend process...
2022-02-10 15:59:46,543 INFO    MainThread:603371 [backend.py:ensure_launched():221] started backend process with pid: 603397
2022-02-10 15:59:46,544 INFO    MainThread:603371 [wandb_init.py:init():469] backend started and connected
2022-02-10 15:59:46,546 INFO    MainThread:603371 [wandb_init.py:init():533] updated telemetry
2022-02-10 15:59:46,561 INFO    MainThread:603371 [wandb_init.py:init():563] communicating current version
2022-02-10 15:59:47,258 INFO    MainThread:603371 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-10 15:59:47,259 INFO    MainThread:603371 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-10 15:59:47,373 INFO    MainThread:603371 [wandb_init.py:init():606] starting run threads in backend
2022-02-10 15:59:49,418 INFO    MainThread:603371 [wandb_run.py:_console_start():1810] atexit reg
2022-02-10 15:59:49,418 INFO    MainThread:603371 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-10 15:59:49,419 INFO    MainThread:603371 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-10 15:59:49,420 INFO    MainThread:603371 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-10 15:59:49,420 INFO    MainThread:603371 [wandb_init.py:init():633] run started, returning control to user process
2022-02-10 15:59:49,420 INFO    MainThread:603371 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-10 15:59:49,420 INFO    MainThread:603371 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-10 15:59:49,421 INFO    MainThread:603371 [wandb_run.py:_restore():1752] restore
2022-02-10 15:59:51,926 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2022-02-10 15:59:52,089 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 759
}

2022-02-10 15:59:52,385 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11575
}

2022-02-10 15:59:52,487 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 759
  total_bytes: 11575
}

2022-02-10 15:59:52,588 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:52,690 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:52,791 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:52,892 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:52,994 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:53,095 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:53,353 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}

2022-02-10 15:59:53,632 INFO    MainThread:603371 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11575
  total_bytes: 11575
}
local_info {
}

2022-02-10 15:59:54,764 INFO    MainThread:603371 [wandb_run.py:_append_files():2180] logging synced files
